lib/ansible/modules/cloud/ovirt/ovirt_vnic_profile.py

```diff
@@ -95,17 +95,6 @@
         value: 9bd9bde9-39da-44a8-9541-aa39e1a81c9d
     network_filter: allow-dhcp
 
-- name: Editing vNICs network_filter, custom_properties, qos
-  ovirt_vnic_profile:
-    name: myvnic
-    network: mynetwork
-    data_center: datacenter
-    qos: myqos
-    custom_properties:
-      - name: SecurityGroups
-        value: 9bd9bde9-39da-44a8-9541-aa39e1a81c9d
-    network_filter: allow-dhcp
-
```

Comment:
Why do we need to remove that example?

Reply:
This is duplicated task. The example is already there.

Reply:
Lines 87-96 duplicate lines 98-107 above. So removing the second of the two duplicate examples makes sense. Thanks @mnecas for catching that!